keyboards/otaku_split/rev0/readme.md

```diff
@@ -0,0 +1,16 @@
+# Otaku Split rev.0
+
+![otaku_split](https://booth.pximg.net/c/620x620/4394ec37-d0ff-4c92-8f78-5c08d0566da6/i/1365150/9953f612-d35f-4f31-873d-2323c7b2f622_base_resized.jpg)
+
+Otaku Split is Japanese layout based keyboard.
+rev.0 is prototype. it has MDF color plate Iron Black and Ayanami blue.
+
+Keyboard Maintainer: [takashiski](https://github.com/takashiski)  
+Hardware Supported: promicro(atmega32u4)
```

Comment:
```suggestion
Hardware Supported: promicro(atmega32u4)  
```

The two spaces are needed at the end here, otherwise this part won't render correctly.